docs/mongocli/command/mongocli-cloud-manager-logs-decrypt.txt

```diff
@@ -0,0 +1,88 @@
+.. _mongocli-cloud-manager-logs-decrypt:
+
+===================================
+mongocli cloud-manager logs decrypt
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Decrypts a log file with the provided local key file or KMIP files.
```

Comment:
Decrypts a Cloud Manager audit log file with the ...

Reply:
see the comment above:
```
this description is shared between OM and CM.
Instead, is this acceptable? (added audit)
"Decrypts an audit log file with the provided local key file or KMIP files.?
```